archive/rpm.ksy

```diff
@@ -0,0 +1,1033 @@
+meta:
+  id: rpm
+  title: RPM Package Manager files
+  file-extension:
+    - rpm
+    - srpm
+    - src.rpm
+    - drpm
+  xref:
+    pronom: fmt/795 # v3
+    wikidata: Q492650
+  license: CC0-1.0
+  ks-version: 0.9
+  endian: be
+doc: |
+  This parser is for the RPM version 3 file format which is the current version
+  of the file format used by RPM 2.1 and later (including RPM version 4.x, which
+  is the current version of the RPM tool). There are historical versions of the
+  RPM file format, as well as a currently abandoned fork (rpm5). These formats
+  are not covered by this specification.
+doc-ref:
+  - https://github.com/rpm-software-management/rpm/blob/master/doc/manual/format.md
+  - https://github.com/rpm-software-management/rpm/blob/master/doc/manual/tags.md
```

Comment:
You should probably have versioned reference here.

Reply:
Urgh, yes, you are correct. Fixed.